client/src/components/application/Transactions.vue

```diff
@@ -164,10 +178,15 @@
 </template>
 
 <script>
+/* eslint-disable */
   import { mapState } from 'vuex'
   import moment from 'moment'
   import CompNameIcon from './Examine/CompNameIcon'
   import Spinner from './spinner'
+  import RequestActionCode from '@/enums/request-action-code'
+  import RequestTypeCode from '@/enums/request-type-code'
+  import EntityTypeCode from '@/enums/entity-type-code'
```

Comment:
How about creating an index file and importing all 3 on the same line?

Reply:
I did try this, but wasn't working quite the same as we we're used too. 
Can see if there is way tho!

Reply:
Yea, i've tried a few things unsuccessfully. Will continue looking however.

Reply:
Don't worry too much about it.